packages/opentelemetry-node/src/spanprocessor.ts

```diff
@@ -201,17 +200,14 @@ function updateSpanWithOtelData(sentrySpan: SentrySpan, otelSpan: OtelSpan): voi
 function updateTransactionWithOtelData(transaction: Transaction, otelSpan: OtelSpan): void {
   const { op, description, source, data } = parseOtelSpanDescription(otelSpan);
 
+  // eslint-disable-next-line deprecation/deprecation
   transaction.setContext('otel', {
```

Comment:
I guess this will generally go away in v8, so no need to worry about this.

Which reminds me, we should also deprecate these I guess...